src/openforms/forms/tests/test_api_forms.py

```diff
@@ -1120,3 +1122,51 @@ def test_detail_shows_translated_values_based_on_cookie(self):
         self.assert_(
             all(literal["value"] == "" for literal in form["literals"].values())
         )
+
+    def test_activate_default_language_translation_enabled_false(self):
+        """
+        Default language should be set, because translation is disabled (ignoring any language headers)
+        """
+        form = FormFactory.create(translation_enabled=False)
+
+        url = reverse(
+            "api:form-set-default-language", kwargs={"uuid_or_slug": form.uuid}
+        )
+        self.client.force_authenticate(user=None)
+        response = self.client.post(url, HTTP_ACCEPT_LANGUAGE="en")
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.data, {"active_language": "nl"})
+        self.assertIn(settings.LANGUAGE_COOKIE_NAME, response.cookies)
+
+        language_cookie = response.cookies[settings.LANGUAGE_COOKIE_NAME]
+
+        self.assertEqual(language_cookie.key, settings.LANGUAGE_COOKIE_NAME)
+        self.assertEqual(language_cookie.value, settings.LANGUAGE_CODE)
```

Comment:
```suggestion
        self.assertEqual(language_cookie.value, "nl")
```
let's make this explicit, if we change the `LANGUAGE_CODE` setting, we'll have weird results and there's misaligned because the `HTTP_ACCEPT_LANGUAGE` header would now suddenly match.

There's a chance in the future we allow configuring the default language via an environment variable.